pyccel/parser/scope.py

```diff
@@ -626,13 +648,18 @@ def get_new_name(self, current_name = None):
         current_name : str, default: None
             The name the user would like to use if possible.
 
+        is_temp : bool, default: False
+            Indicates if the generated symbol should be a temporary (i.e. an extra
+            temporary object generated by Pyccel). This is always the case if no
+            current_name is provided.
```

Comment:
I find it quite strange that the default value `is_temp = False` clashes with the default value `current_name = None`. Is there no way to avoid this?

Reply:
I could make the default `None`?

Reply:
101a564071338b0e56e785e95159ccf09bfed07c